dapp/src/hooks/useScrollbarVisibility.ts

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useEffect, useState } from "react"
+
+const SCROLLBAR_WIDTH = `${window.innerWidth - document.body.offsetWidth}px`
+
+function scrollbarVisible(className: string) {
+  const element = document.getElementsByClassName(className)[0]
+  return element?.scrollHeight > element?.clientHeight
+}
+
+export default function useScrollbarVisibility(className: string) {
+  const [isVisible, setIsVisible] = useState(false)
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsVisible(scrollbarVisible(className))
+    }
+    window.addEventListener("resize", handleResize)
+
+    return () => {
+      window.removeEventListener("resize", handleResize)
+    }
+  }, [className])
+
+  const refetch = useCallback(() => {
```

Comment:
Non blocking:
Fetching in computer science has numerous meanings. Generally speaking it describes the process of gathering data. The `refetch` name made me confused. The first what i look for when I see term `fetch` is the call of `axios` or `fetch` functions. I'd suggest to change it to something more descriptive like `handleScrollbarVisibilityCheck`. What do you think?

Reply:
This function checks and sets the state again `isVisible`. So let's change its name simply to `refreshState`.